tests/mariner2/install.sh

```diff
@@ -30,8 +27,7 @@ install() {
         moby-containerd \
         moby-engine \
         moby-cli \
-        moby-compose \
-        moby-init
+        moby-compose
```

Comment:
Why do these not have versions, but the ones in `tests/centos8/install.sh` do?

Reply:
Oh good catch.